demo/screengrabs/Makefile

```
@@ -1,52 +1,20 @@
+.DEFAULT_GOAL := help
 
-build-all:
-	@./build-all.sh demo-stacks.txt
-	@echo "Validating screengrabs..."
-	@! grep -r "atmos: not found" artifacts/ || (echo "ERROR: Found 'atmos: not found' in screengrabs" && exit 1)
-	@! grep -r "COMMAND_EXIT_CODE" artifacts/ || (echo "ERROR: Found 'COMMAND_EXIT_CODE' in screengrabs" && exit 1)
-	@! grep -r "Script started" artifacts/ || (echo "ERROR: Found 'Script started' messages in screengrabs" && exit 1)
-	@! grep -r "Script done" artifacts/ || (echo "ERROR: Found 'Script done' messages in screengrabs" && exit 1)
-	@echo "✓ All screengrabs validated successfully"
+.PHONY: help $(MIGRATION_TARGETS)
+help:
```

checkmake (0.3.2)

[warning] Target body for "help" exceeds allowed length of 5 lines (10).

(maxbodylength)